<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="rootLayout">
   <property name="styleSheet">
    <string notr="true">background-color:#3FAB21</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>16</number>
    </property>
    <property name="topMargin">
     <number>16</number>
    </property>
    <property name="rightMargin">
     <number>16</number>
    </property>
    <property name="bottomMargin">
     <number>16</number>
    </property>
    <item row="5" column="0">
     <widget class="QTextEdit" name="etNotes">
      <property name="styleSheet">
       <string notr="true">background:#f2f2f2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblStartedWorkAt">
      <property name="font">
       <font>
        <family>Rubik</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#fff</string>
      </property>
      <property name="text">
       <string>Started work at: 01:10</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblTimer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Rubik</family>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#fff</string>
          </property>
          <property name="text">
           <string>01 : 33 : 55</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblWorkedFor">
          <property name="font">
           <font>
            <family>Rubik</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#fff</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblBreakFor">
          <property name="font">
           <font>
            <family>Rubik</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#fff</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>16</number>
        </property>
        <property name="leftMargin">
         <number>16</number>
        </property>
        <property name="topMargin">
         <number>16</number>
        </property>
        <property name="rightMargin">
         <number>16</number>
        </property>
        <property name="bottomMargin">
         <number>16</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnStart">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Rubik</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#fff</string>
          </property>
          <property name="text">
           <string>START</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBreak">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Rubik</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#fff; background-color:#D72638;</string>
          </property>
          <property name="text">
           <string>TAKE A BREAK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLog">
          <property name="font">
           <font>
            <family>Rubik</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#fff; background-color:#FB8B24;</string>
          </property>
          <property name="text">
           <string>VIEW LOG</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lblToday">
      <property name="font">
       <font>
        <family>Rubik</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#fff</string>
      </property>
      <property name="text">
       <string>01/01/2021</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Rubik</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#fff</string>
      </property>
      <property name="text">
       <string>Notes</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>348</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
